README.md

```diff
@@ -178,72 +192,46 @@ OS X:
 }];
 ```
 
-### xAuth
+##### xAuth
 
-Please note that xAuth access
-[must be specifically requested](http://www.tumblr.com/oauth/apps)
-for your application.
+Please note that xAuth access is no longer available to 3rd party developers
```

Comment:
Could be nice to explain why or link to something that explains transition plan for people currently using xauth.